go/database/mpt/tool/export_test.go

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package main
+
+import (
+	"context"
+	"errors"
+	"io"
+	"path/filepath"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/Fantom-foundation/Carmen/go/common"
+	"github.com/Fantom-foundation/Carmen/go/database/mpt"
+	cio "github.com/Fantom-foundation/Carmen/go/database/mpt/io"
+)
+
+func TestExport_CanBeInterrupted(t *testing.T) {
```

Comment:
I now noticed that this test should probably go to the IO package, it tests methods from that package, not the Tool package.

Reply:
Yea I agree. I thought it would create unnecessary duplicity, but the duplicity is very small given the fact the code was a bit complicated with the interrupt check inside `tool` package. Moved to `io` package.